atom/browser/api/atom_api_power_monitor.cc

```diff
@@ -38,8 +39,14 @@ void PowerMonitor::OnResume() {
 }
 
 // static
-mate::Handle<PowerMonitor> PowerMonitor::Create(v8::Isolate* isolate) {
-  return CreateHandle(isolate, new PowerMonitor);
+v8::Handle<v8::Value> PowerMonitor::Create(v8::Isolate* isolate) {
+  if (!Browser::Get()->is_ready()) {
+    node::ThrowError(R"(Cannot initialize "power-monitor" module
```

Comment:
oh didnt know that, went by cpplint suggestion at the time. Would be nice if `cpplint` was modified to follow chromium styleguide :)